recipe-client-addon/test/browser/browser_FilterExpressions.js

```diff
@@ -47,8 +61,32 @@ add_task(async function() {
   is(val, true, "Bucket sample returns true for a known sample");
   val = await FilterExpressions.eval('["test-4"]|bucketSample(0, 5, 10)');
   is(val, false, "Bucket sample returns false for a known sample");
+});
 
-  // Test that it reads from the context correctly.
-  val = await FilterExpressions.eval("first + second + 3", {first: 1, second: 2});
-  is(val, 6, "context is available to filter expressions");
+// Preference tests
+add_task(async function() {
+  let val;
+  // Compare the value of the preference
+  Preferences.set("normandy.test.value", 3);
```

Comment:
This should use [`SpecialPowers`](https://github.com/mozilla/normandy/blob/6e96cacba67546909c37c7e574c694042b0267d0/recipe-client-addon/test/browser/browser_ClientEnvironment.js#L31) to set the preference, which makes sure the preference gets cleaned up after the test.